morango/constants/settings.py

```diff
@@ -2,6 +2,7 @@
 MORANGO_SERIALIZE_BEFORE_QUEUING = True
 MORANGO_DESERIALIZE_AFTER_DEQUEUING = True
 MORANGO_DISALLOW_ASYNC_OPERATIONS = False
+MORANGO_INSTANCE_INFO = {}
```

Comment:
Did you decide against making it an import path and just having a bare dictionary here?